Azure/sonic-buildimage
# [openssh] Introduce custom openssh-server package for supporting reverse console SSH

**- Why I did it**

To support reverse SSH feature, we need to build a custom openssh-server package and replace the official one in SONiC OS.

(see HLD for more detail: https://github.com/Azure/SONiC/blob/master/doc/console/SONiC-Console-Switch-High-Level-Design.md#313-reverse-ssh)

**- How I did it**

- Modify source code of openssh-server to allow the sshd expose the `style` field in `$SSH_TARGET_CONSOLE_LINE`.
- Build custom openssh-server
- Install custom openssh-server

**- How to verify it**

- [x] Build custom openssh-server package and install/test it on a physical SONiC switch
- [x] Build an image which includes the change of current PR and install/test it on a physical DUT

Test Steps:
1. Login with below options
![image](https://user-images.githubusercontent.com/6898708/97276410-1ace5400-1872-11eb-880c-638069294ace.png)

1. Run below command and check the result
```bash
admin@sonic:~$ echo $SSH_TARGET_CONSOLE_LINE
1
```

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**
